### ISSUE TITLE
Disable cache on WP query to save memory

### ISSUE THREAD

**Issue by julienbourdeau** [algolia/algoliasearch-wordpress#627]
### What did you expect to happen?

If the memory is limited your might get a fatal error. Apparently, a good way around this is to disable the object cache. [Here](https://github.com/algolia/algoliasearch-wordpress/blob/master/includes/indices/class-algolia-searchable-posts-index.php#L301-L305), you can pass an option for it.

```php
$query = new WP_Query( array(
  'post_type' => $this->post_types,
  'post_status' => 'any', // Let the `should_index` take care of the filtering.
  'suppress_filters' => true,
  'cache_results' => false // <---- here
) );

```

See [support ticket](https://secure.helpscout.net/conversation/388641729/31190/) for further details.

### Technical info
* WordPress version: 2.7.5
* Algolia Search plugin version: 2.4

**Comment by rayrutjes** [comment]
This will be done with #626

**Comment by julienbourdeau** [comment]
I checked through issue but I didn't look at PRs ^^'

**Comment by rayrutjes** [comment]
This is now merged in: #633 and will be released soon.
